### PR TITLE
Update metaspades.xml

### PR DESCRIPTION
**Pull request by pravs3683**
PR to output Assembly-graph FASTG file in the Galaxy history. Currently, there is no "fastg" format defined on Galaxy (If I am wrong, please let me know), so, I have used regular "text" format to output the fastg file.

Thanks,
Praveen

FOR CONTRIBUTOR:
* [x ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

FOR REVIEWER:
* [ ] .shed.yml file ok
    - [ ] Toolshed user `iuc` has access to associated toolshed repo(s)
* [ ] Indentation is correct (4 spaces)
* [ ] Tool version/build ok
* [ ] `<command/>`
  - [ ] Text parameters, input and output files `'single quoted'`
  - [ ] Use of `<![CDATA[ ... ]]>` tags
  - [ ] Parameters of type `text` or having `optional="true"` attribute are checked with `if str($param)` before being used
* [ ] Data parameters have a `format` attribute containing datatypes recognised by Galaxy
* [ ] Tests
  - [ ] Parameters are reasonably covered
  - [ ] Test files are appropriate
* [ ] Help
  - [ ] Valid restructuredText and uses `<![CDATA[ ... ]]>` tags
* [ ] Complies with other best practice in [Best Practices Doc](http://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html)

### CODE REVIEW

**Comment by bgruening** [comment]
@pravs3683 are you up to add a test for this output? If you like we can also help you to add this datatype to Galaxy.

Thanks for your contribution!

**Comment by pravs3683** [comment]
I am not sure how to add the test to this output. The fastg output is being
generated by the tool but is lost because it is not mentioned in the
wrapper. I am generating the output for fastg using test-data shared in the
repository, once done, I will share it on GitHub repository. Additionally,
will add test output tag in the wrapper, and then I will create a new pull
request.

In the tests, I see only the output tag under tests tag, only for contig
output. Should I add the tag for the scaffolds and fastg both?

**Comment by pravs3683** [comment]
The Travis test failed on the some Intermine tool (see the error log from Travis). I don't know if that is relevant to Metaspades or not: 

+Linting tool /home/travis/build/galaxyproject/tools-iuc/tools/intermine_galaxy_exchange/intermine_galaxy_exchange.xml
Applying linter tests... FAIL
.. ERROR: Found output tag with unknown name [tabular_output], valid names [['intermine_output']]
Applying linter shed_metadata... WARNING
.. WARNING: Missing shed metadata field [homepage_url] for repository
Failed linting

File: tools/metaspades/metaspades.xml [hunk 1]
```diff
@@ -72,6 +72,7 @@
     <outputs>
         <data name="out_contigs" format="fasta" from_work_dir="contigs.fasta" label="SPAdes contigs (fasta)" />
         <data name="out_scaffolds" format="fasta" from_work_dir="scaffolds.fasta" label="SPAdes scaffolds (fasta)" />
+        <data name="out_fastg" format="text" from_work_dir="assembly_graph.fastg" label="SPAdes assembly graph (fastg)" />
```
Comment (nsoranzo) [inline tools/metaspades/metaspades.xml]: You should use `txt` not `text` here (if you're not going to add the new datatype @bgruening mentioned).

**Comment by nsoranzo** [comment]
Restarting the Travis build fixed the linting error (weird!).

**Pull request closed without merging**